pkgs/tools/system/journalwatch/default.nix

```diff
@@ -1,15 +1,16 @@
-{ stdenv, buildPythonPackage, fetchurl, pythonOlder, systemd, pytest }:
+{ stdenv, buildPythonPackage, fetchFromGitHub, pythonOlder, systemd, pytest }:
 
 buildPythonPackage rec {
   pname = "journalwatch";
   name = "${pname}-${version}";
```

Comment:
While you're in here cleaning up would you mind removing this as it is no longer required?

Reply:
Good point, adressed.